coralproject/talk
# [CORL-2515] update pr template to ask about migrated tests

## What does this PR do?
This PR updates our PR template to include a section for indicating any frontend tests that have been migrated to our new testing framework.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
Open a PR against this branch (double checking this) and observe that the new section is present.
 
 
## How do we deploy this PR?
No special considerations should be necessary.
